code-challenges.test.js

```diff
@@ -14,19 +14,58 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// RED
+describe ("fibonacci", () => {
+  it ("takes in a number greater than 2", () => {
+    expect (fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect (fibonacci(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// GREEN
+// declare a function (fibonacci) that takes in a number greater than two (which represents the number of values in the array) and returns an array of numbers (starting at one) that follow the Fibonacci sequence
+// use a for loop to iterate a specified number of times following a specific set of actions
+// create an empty array to push (using .push()) each iteration into
+// specify that the next number will be the added to the last number (using the index to specify)
+// input: a number greater than two 
+// output: an array of numbers following the Fibonacci sequence
+const fibonacci = (num) => {
+  let arr = [1,1]
+
+  for (let i = 1; i < num -1; i++) {
+    arr.push(arr[i] + arr[i - 1])
+  }
+
+  return arr
+}
+console.log(fibonacci(fibLength1))
+// output: [1, 1, 2, 3, 5, 8]
+console.log(fibonacci(fibLength2))
+// output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+ // note: I was stuck on this for a long time, overthinking how to incorporate the index, but using help from a classmate I was able to understand this method. I initially was not creating an array to push new numbers into, nor was I even pushing values. Thanks to my peers and the resources I have available, I was able to come to an understanding on how to execute this code.
```

Comment:
As always, you are so good at recognizing all of the moving pieces that were happening while you were writing this code. Enough so that you can elaborate it perfectly. Great job!